.rubocop.yml

```diff
@@ -1 +1,25 @@
-inherit_from: .rubocop_rspec_base.yml
+inherit_from:
+  - .rubocop_rspec_base.yml
+
+Metrics/AbcSize:
+  Max: 28
+
+Metrics/BlockLength:
+  Max: 86
+
+Metrics/PerceivedComplexity:
+  Max: 10
+
+Security/MarshalLoad:
+  Exclude:
+    - 'lib/rspec/support/spec/in_sub_process.rb'
+
+Style/EvalWithLocation:
+  Exclude:
+    # eval is only used here to check syntax
+    - 'lib/rspec/support/ruby_features.rb'
+
+Lint/AssignmentInCondition:
+  Exclude:
+    # The pattern makes sense where
```

Comment:
I don't understand this comment.  Looks like you intended to write more after "where"?

Reply:
lol, `s/where/here/`